pkgs/servers/coturn/default.nix

```diff
@@ -1,17 +1,24 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, openssl, libevent }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, openssl
+, libevent
+, pkgconfig
+}:
 
 stdenv.mkDerivation rec {
   pname = "coturn";
-  version = "4.5.1.3";
+  version = "4.5.2";
 
   src = fetchFromGitHub {
     owner = "coturn";
     repo = "coturn";
     rev = version;
-    sha256 = "1801931k4qdvc7jvaqxvjyhbh1xsvjz0pjajf6xc222n4ggar1q5";
+    sha256 = "1s7ncc82ny4bb3qkn3fqr0144xsr7h2y8xmzsf5037h6j8f7j3v8";
   };
 
-  buildInputs = [ openssl libevent ];
+  buildInputs = [ openssl pkgconfig libevent ];
```

Comment:
```suggestion
  buildInputs = [ openssl pkg-config libevent ];
```

Reply:
`pkg-config` should be in nativeBuildInputs. Also please add `libprom` to here.

Reply:
Also adding more libs for prometheus support.